pkg/logger/logger_test.go

```diff
@@ -0,0 +1,31 @@
+package logger_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/surrealdb/surrealdb.go/pkg/logger"
+)
+
+func TestLog(t *testing.T) {
+	templogger, err := logger.CreateLogFile("../../surrealdb.log")
```

Comment:
This would be a perfect example of using an in memory logger as you can validate what was written to it without writing to files.

Reply:
done.